galaxy.yml

```diff
@@ -0,0 +1,102 @@
+### REQUIRED
+# The namespace of the collection. This can be a company/brand/organization or product namespace under which all
+# content lives. May only contain alphanumeric lowercase characters and underscores. Namespaces cannot start with
+# underscores or numbers and cannot contain consecutive underscores
+namespace: cifmw
+
+# The name of the collection. Has the same character restrictions as 'namespace'
+name: general
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+# The path to the Markdown (.md) readme file. This path is relative to the root of the collection
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
```

Comment:
probably stupid, but maybe ordering this list with alphabetical order would be better?

Reply:
sounds good, I just got the list from the repo contributors, will update in the next version